superdesk/media/image.py

```diff
@@ -44,8 +56,13 @@ def get_meta(file_stream):
                 value = {ExifTags.GPSTAGS[vk].strip(): vv for vk, vv in v.items()}
                 exif_meta[key] = value
             else:
-                value = v.decode('UTF-8') if isinstance(v, bytes) else v
-                exif_meta[key] = value
```

Comment:
@nistormihai you're right about the placement of the method. I change its place. 
The new PR is available on the `superdesk/superdesk-core` repo:
https://github.com/superdesk/superdesk-core/pull/411